src/irmin-unix/resolver.ml

```diff
@@ -701,7 +716,7 @@ let remote () =
       headers str =
     let y = read_config_file config_path in
     let store =
-      build_irmin_config y root opts (store, hash, contents) branch commit
+      build_irmin_config y root opts (store, hash, contents) branch commit None
```

Comment:
can't we in this case too, get the plugin from the config file?

Reply:
Yes, that's done in `get_store` (https://github.com/mirage/irmin/blob/fb529e170e4d34c350dc9106fc0494722d1ea1e3/src/irmin-unix/resolver.ml#L507), so setting the plugin in a config file works, just like other options the command line should override settings from the config file.

Reply:
ah, yes, thanks!